pkg/controller/node_watcher.go

```diff
@@ -172,7 +171,7 @@ func (watcher *NodeWatcher) WatchNodes() {
 			return false
 		}
 		if !errors.IsNotFound(err) {
-			klog.V(2).Infof("error getting node %q from informer: %v", key, err)
+			klog.V(2).ErrorS(err, "Error getting node from informer", "node", key)
```

Comment:
Can you keep this at InfoS?

Reply:
Sorry for the mistake. I've updated it to use InfoS.

Reply:
@xing-yang Sorry for the mistake. I've updated it to use InfoS.